config/i18n.json

```diff
@@ -1029,6 +1029,8 @@
       "CustomTotal": "Custom Total",
       "MasterworkTier": "Masterwork Tier",
       "MasterworkStat": "Masterwork Stat",
+      "DeepsightLevel": "Deepsight Level",
```

Comment:
This could be just "Level", since there aren't other types of Level for weapons (so far)